### ISSUE TITLE
Website Accessibility-WCAG 3.1.1: Ensures every HTML document has a lang attribute (html)

### ISSUE THREAD

**Issue by A11Ybot-OP** [SFTtech/openage#1453]
#### Issue
`Ensures every HTML document has a lang attribute` ([html-has-lang](https://accessibilityinsights.io/info-examples/web/html-has-lang))
#### Target application
[openage - free, modern RTS game engine](https://openage.sft.mx/)
#### Element path
html
#### Snippet
`<html>`
#### How to fix
    Fix any of the following:
      The <html> element does not have a lang attribute
#### Environment
Microsoft Edge version 96.0.1054.62
